docs/docs/build/metrics-view/what-are-metrics-views.md

```diff
@@ -0,0 +1,129 @@
+
+[Customizing](/build/metrics-view/customize) your time series...
+
+### Measures
+
+Measures are numeric aggregates of columns from your data model. A measure must be defined with [DuckDB SQL](https://duckdb.org/docs/sql/introduction.html) aggregation functions and expressions on columns from your data model. The following operators and functions are allowed in measure expressions:
+
+* Any DuckDB SQL [numeric](https://duckdb.org/docs/sql/functions/numeric) operators and functions
+* This set of DuckDB SQL [aggregates](https://duckdb.org/docs/sql/aggregates): `AVG`, `COUNT`, `FAVG`, `FIRST`, `FSUM`, `LAST`, `MAX`, `MIN`, `PRODUCT`, `SUM`, `APPROX_COUNT_DISTINCT`, `APPROX_QUANTILE`, `STDDEV_POP`, `STDDEV_SAMP`, `VAR_POP`, `VAR_SAMP`.
+* [Filtered aggregates](https://duckdb.org/docs/sql/query_syntax/filter.html) can be used to filter the set of rows fed to the aggregate functions.
+
+As an example, if you have a table of sales events with the sales price and customer ID, you could calculate the following metrics with these aggregates and expressions:
+* Number of sales: `COUNT(*)`
+* Total revenue: `SUM(sales_price)` 
+* Revenue per customer: `CAST(SUM(sales_price) AS FLOAT)/CAST(COUNT(DISTINCT customer_id) AS FLOAT)`
+* Number of orders with order value more than $100: `count(*) FILTER (WHERE order_val > 100)`
+
+You can also add labels, descriptions, and your choice of number formatting to customize how they are shown. See our dedicated examples and pages for the following advanced measures!
+- **[Metric Formatting](/build/metrics-view/metric-formatting)**
+- **[Case Statements and Filters](/build/metrics-view/case-statements)**
+- **[Referencing Measures](/build/metrics-view/referencing)**
+- **[Quantiles](/build/metrics-view/quantiles)**
+- **[Fixed Metrics](/build/metrics-view/fixed-metrics)**
+- **[Window Functions](/build/metrics-view/windows)**
+
+  
+
+### Dimensions
+
+Dimensions are used for exploring segments and filtering. Valid dimensions can be any type and are selected using the drop-down menu. You can also add labels and descriptions to your dimensions to customize how they are displayed. See our dedicated examples and pages for more use cases.
+
+- **[Clickable Dimension Links](/build/metrics-view/dimension-uri)**
+- **[Unnest Dimensions](/build/metrics-view/unnesting)**
+- **[Lookups](/build/metrics-view/lookup)**
+
+## Security
+
+Data access is an important part of Rill that allows you to create specified views of your dashboard depending on who's viewing the page. For a dedicated guide, see [Data Access](/build/metrics-view/security) for more information.
+
+
+## Multi-Editor and External IDE Support
```

Comment:
This isn't specific to Metrics Views. What if this were a dedicated page at the top-level of the "Build" docs, immediately prior to the "Debugging in Rill Developer" section?